pixi.toml

```diff
@@ -101,7 +101,7 @@ cpp-prepare-msvc = "cmake -G 'Visual Studio 17 2022' -B build-msvc -S ."
 [dependencies]
 attrs = ">=23.1.0"
 blackdoc = "0.3.8"
-clang-tools = ">=15,<16"     # clang-format
+clang-tools = "16.0.6"       # clang-format
```

Comment:
Looks like this is maybe tickling a new lint error: https://github.com/rerun-io/rerun/actions/runs/7821224421/job/21337710444?pr=5102

Reply:
urgh. yeah I should undo all that stuff. the real fix was putting the codegen run on pixi. then again would be nice to run against newer clang, but maybe not in here!

Reply:
... let's first try the forward-flight!